content/topics/kubernetes.md

```diff
@@ -6,265 +6,141 @@ url: /kubernetes
-          can then be used to deploy apps to.
-      code: |
-          import * as gcp from "@pulumi/gcp";
-          import { nodeCount, nodeMachineType, password, username } from "./config";
-
-          const engineVersion = gcp.container.getEngineVersions().then(v => v.latestMasterVersion);
-          export const k8sCluster = new gcp.container.Cluster("gke-cluster", {
-              initialNodeCount: nodeCount,
-              minMasterVersion: engineVersion,
-              nodeVersion: engineVersion,
-              masterAuth: { username, password },
-              nodeConfig: {
-                  machineType: nodeMachineType,
-                  oauthScopes: [
-                      "https://www.googleapis.com/auth/compute",
-                      "https://www.googleapis.com/auth/devstorage.read_only",
-                      "https://www.googleapis.com/auth/logging.write",
-                      "https://www.googleapis.com/auth/monitoring"
-                  ],
-              },
-          });
-
-      cta:
-          url: /docs/get-started
-          label: GET STARTED
+        Pulumi is the modern platform to manage all of your cloud native infrastructure using familiar engineering tools and workflows. Avoid complex YAML by using your favorite programming languages and automate your deployments to Amazon EKS, Azure AKS, Google GKE, DigitalOcean DOKS, multi-cloud, hybrid and on-premises clusters with leading ecosystem integrations.
+
+video_section:
+  title: Pulumi In Action
+  subtitle: Watch how easy it is to setup Amazon EKS in 5 minutes with Pulumi
+  youtube_video_id: yA40w1ryMu8
+  video_title: Get Started with Amazon EKS in 5 Minutes
+
+kubernetes_overview:
+    title: Cloud Native Engineering with Pulumi
+    description: |
+        Pulumi streamlines Kubernetes cluster configuration, management, and app workload deployments to your clusters.
+
+        With Pulumi you can:
+
+    list:
+        - Provision Kubernetes clusters on all major cloud providers.
+        - Increase productivity using the full ecosystem of dev tools such as IDE auto-completion, type & error checking, linting, refactoring, and test frameworks to validate Kubernetes clusters, app workloads, or both.
+        - Automate deployments with CI/CD integrations for [Spinnaker](https://www.pulumi.com/blog/unlocking-spinnaker-with-pulumi/), [Octopus](https://www.pulumi.com/blog/deploying-with-octopus-and-pulumi/), [GitHub Actions](https://www.pulumi.com/blog/continuous-delivery-to-any-cloud-using-github-actions-and-pulumi/), [GitLab](https://www.pulumi.com/blog/continuous-delivery-with-gitlab-and-pulumi-on-amazon-eks/), [Azure DevOps](https://www.pulumi.com/blog/cd-made-easy-with-pulumi-and-azure-pipelines/) and [more](https://www.pulumi.com/docs/guides/continuous-delivery/).
+        - Seamlessly manage cloud resources with the Pulumi Kubernetes Operator.
+        - And more...
+
+    cta: REQUEST MORE INFORMATION
```

Comment:
Text size is larger on this CTA than other CTAs below and above.  Can we standardize these @cnunciato

Reply:
Already done, yep.